src/Symfony/Bundle/FrameworkBundle/Tests/Command/EventDispatcherDebugCommandCompletionTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Command\EventDispatcherDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandCompletionTester;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class EventDispatcherDebugCommandCompletionTest extends TestCase
+{
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        $tester = $this->createCommandCompletionTester();
```

Comment:
Did you think about only mocking the `ContainerInterface` here (so no kernel, no application)? 

```php
$dispatchers = $this->createMock(ContainerInterface::class);
$dispatchers->method('get')->willReturn($dispatcher);
$dispatchers->method('has')->willReturn(true);
// ...

$command = new EventDispatcherDebugCommand($dispatchers);
$tester = new CommandCompletionTester($command);
```

Reply:
Let me try this out

Reply:
But container interface don't have another method required
`getServiceIds`, `getAliases`
Will mocking this interface able to make method for methods not on the interface?